pyuvdata/uvdata/mir_meta_data.py

```diff
@@ -2046,7 +2171,7 @@ def _add_check(self, other, merge=None, overwrite=None, discard_flagged=False):
 
         return this_idx, other_idx, this_mask[this_idx], other_mask[other_idx]
 
-    def read(self, filepath):
+    def read(self, filepath=None):
```

Comment:
So pyuvdata is generally moving towards getting rid of positional arguments in favor of keywords. This would nominally be the exception, except that there are subclasses where there are multiple arguments, so to keep things consistent I've made it a keyword here. I did add a catch for `None` though in be15abab6f4548a576351a1f43d190cefd2fcd48.